packages/cells/src/widget.ts

```diff
@@ -1293,8 +1295,28 @@ export class CodeCell extends Cell<ICodeCellModel> {
     if (this.syncScrolled) {
       this.saveScrolledState();
     }
+    this.updateOutputIcon();
   }
 
+  /**
+   * Update the Propmpt Overlay Icon
```

Comment:
```suggestion
   * Update the Prompt Overlay Icon
```

Should the method name include word "prompt" too?

Reply:
Upated to `updatePromptOverlayIcon`.